config/scheduler/settings.ini

```diff
@@ -10,6 +10,9 @@ pa.scheduler.home=.
 # When the server has a public endpoint, different from the hostname available on the machine, this property should be used to correctly set the url
 #pa.scheduler.rest.url=
 
+#Catalog rest endpoint
+pa.catalog.rest.url=http://localhost:8080/catalog
```

Comment:
Maybe there is some work to do in the class SchedulerStarter to makes configuration easier. Currently the rest url is inferred dynamically (if not set explicitly) the catalog could also be.